src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar_js.html.twig

```diff
@@ -14,6 +14,11 @@
             'sfwdt{{ token }}',
             '{{ path("_wdt", { "token": token }) }}',
             function(xhr, el) {
+
+                /* Evaluate embedded scripts inside the toolbar */
+                var rx=/<script>(.*?)<\/script>/g, s;
+                while (s = rx.exec(el.innerHTML)) {eval(s[1]);};
```

Comment:
actually, your CSP handler does not inject the rule allowing eval, so this looks entirely broken.